src/ProxyClient/Dispatcher.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\ProxyClient;
+
+use FOS\HttpCache\Exception\ExceptionCollection;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Queue and send HTTP requests with a Httplug asynchronous client.
```

Comment:
this needs to be adjusted. maybe just "Queue and send HTTP invalidation requests."